features/step_definitions/receive-steps.js

```diff
@@ -0,0 +1,18 @@
+import { Given, When, Then } from 'cucumber';
+import { expect } from 'chai';
+import { waitAndClick, waitAndSelect } from '../support/helpers/shared-helpers';
+
+Given('I generate {int} addresses', async function (numberOfAddresses) {
+  for (let i = 0; i < numberOfAddresses; i++) {
+    await waitAndClick(this.client, '.generateAddressButton:not(.WalletReceive_spinning)');
+  }
+});
+
+When('I click the ShowUsed switch', async function () {
+  await waitAndClick(this.client, '.SimpleSwitch_switch');
+});
+
+Then('I should see {int} addresses', async function (numberOfAddresses) {
+  const addresses = await waitAndSelect(this.client, '.WalletReceive_walletAddress', '.generatedAddress-' + numberOfAddresses, 50000);
```

Comment:
@daniloprates please remove the `50000` timeout param in this call. We need to always use the default one.

Reply:
Also here instead of `waitAndSelect` use the helper I described in my next comment: `getVisibleElementsForSelector `

Reply:
PS. please use `.generatedAddress-${numberOfAddresses}` instead of `'.generatedAddress-' + numberOfAddresses` in the call...